FX/bryanchi_FX Devices/FXD Macros.jsfx

```diff
@@ -0,0 +1,333 @@
+noindex: true
+
+version: 1.1.0
+desc:Macros
+noindex: true
```

Comment:
This noindex is duplicated. The package editor already adds it for you at the very top of the file.